wikimedia/wikipedia-ios-components
# Add Watchlist Filters components

**Phabricator:** 
https://phabricator.wikimedia.org/T342929

### Notes
This is the UI piece to Watchlist Filters. The big new component here is the `WKFormView` (open to a name change here!) and related classes. For now it only has the ability to display a list of single-select or multi-select sections, and updates related view models.

Note this is dependent on https://github.com/wikimedia/wikipedia-ios-data/pull/6 & https://github.com/wikimedia/wikipedia-ios-data/pull/9.

### Test Steps
Load the watchlist filter view. Confirm it themes properly and elements scale with dynamic type. End-to-end testing will need to be done in the client app with an upcoming PR.
